Translator/Extractor/JMSTranslatorBundle/AdminExtractor.php

```diff
@@ -22,7 +22,11 @@
 use Symfony\Component\HttpKernel\Log\LoggerInterface;
 use Symfony\Component\Translation\TranslatorInterface;
 
-class AdminExtractor implements ExtractorInterface, TranslatorInterface, SecurityHandlerInterface, LabelTranslatorStrategyInterface
+final class AdminExtractor implements
+    ExtractorInterface,
```

Comment:
BTW, the goal of this PR is to close this class by making it final, let keep the final diff minimal.